Data-Analytics/Spark-S3-secret/spark-secret-creation.ipynb

```diff
@@ -1,26 +1,30 @@
 {
```

Comment:
I'm not sure that we need this in general, as since 3.5.0 release we added custom EzSparkAWSCredentialProvider which read authentication token automatically.

To enable that provider user need to add the following properties to Spark Application configuration:
```
spark.hadoop.fs.s3a.aws.credentials.provider: org.apache.spark.s3a.EzSparkAWSCredentialProvider
spark.hadoop.fs.s3a.endpoint: "http://local-s3-service.ezdata-system.svc.cluster.local:30000/"
spark.hadoop.fs.s3a.path.style.access: true
spark.hadoop.fs.s3a.connection.ssl.enabled: false
```

I mean, if we are talking about our custom S3 object store server.
If the goal is to access AWS S3, everything remains the same.